compiler/src/iree/compiler/Dialect/Stream/Transforms/test/encode_host_tensors_packing.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt --split-input-file --iree-stream-encode-host-tensors --verify-diagnostics %s | FileCheck %s
+// RUN: iree-opt --split-input-file --iree-stream-encode-host-tensors --iree-experimental-packed-i1-storage --verify-diagnostics %s | FileCheck %s
```

Comment:
I think it is better to create a new file for the experimental pakced i1 tests. Otherwise we're missing the test coverage for default path. Once we flip the behavior to default, we can update the tests.

Reply:
I updated the code to exclude any changes and only apply the change to `i1` type. so this change is reverted.

Reply:
I don't get where is the flag getting tested? It is only covered by an e2e test, but we should have a pass level test IMO.